### ISSUE TITLE
Sign in with Google temporarily disabled for this app

### ISSUE THREAD

**Issue by prhbrt** [googleworkspace/md2googleslides#82]
When using the app for the first time and going through the authorization steps, google says:

> Sign in with Google temporarily disabled for this app
> 
> This app has not been verified yet by Google in order to use Google Sign In.

When authenticating, Google complained they didn't check the app and hence it's disabled.

I fixed this by going to the [Google Cloud Console](https://console.developers.google.com/apis/credentials), I guess you have to be a developer to have this.

Then I created a project 'md2googleslides' and I created a 'OAuth Client ID' for 'Desktop app' also with the name 'md2googleslides', this gave me a client ID and client Secret.

The ID and the secret I manually injected into `bin/md2gslides`, in particular `/home/herbert/.node_modules_global/bin/md2gslides` for my installation. Around line 130, in the `authorizeUser()` function, there are `clientId` and `clientSecret` entries.

I think this should work, but I'm still working some stuff out and it is an ugly work around.

**Comment by wescpy** [comment]
Yep, correct. This is a duplicate of </issues/70> which was closed in favor of </issues/64>. My fix is almost exactly what you suggest but cleaner as it avoids that "ugly workaround" as you suggested. The PR is open for your review at </pull/74>.

**Comment by sprajagopal** [comment]
Thanks for this amazing project! It was super useful to me.
I was trying this work around for the sign-in process. It works sometimes but not always. Particularly the rate-limiting on the number of queries is a problem.
Anyway, I tried a few options but nothing that works for me and obviously this is dependent on google heavily (and that's obviously the point of the project, I get that)
In case, anyone is looking for a simpler way to do this, `pandoc-2.10.x` now can work seamlessly in converting markdown to pptx. You can then use simple google drive sync. This is the best I could find that can emulate the above without the GSuite problems.

**Comment by wescpy** [comment]
I provided a [temporary solution](/issues/70#issuecomment-616432383) for those who needs this working ASAP and filed a [PR with a permanent fix](/pull/74), but that's undergoing review at this time.

**Comment by jpoehnelt** [comment]
Closing this issue as the repository is archived.
